pyupgrade.py

```diff
@@ -1789,13 +1857,15 @@ def _replace(i, mapping, node):
             if any(tok.name == 'NL' for tok in tokens[i:end]):
                 continue
             _replace_call(tokens, i, end, func_args, '{args[0]}')
-        elif token.offset in visitor.simple_ids:
-            tokens[i] = Token(
-                token.name,
-                'OSError',
-                token.line,
-                token.utf8_byte_offset,
-            )
+        elif token.offset in visitor.os_error_alias_calls:
+            node = visitor.os_error_alias_calls[token.offset]
+            if isinstance(node.func, ast.Name):
```

Comment:
hmmm you end up with `Name`s in the `_calls` mapping? shouldn't that be in the `alias_simple` mapping instead

Reply:
I think I end up parsing the `Name` that's inside the `Call` :/ will think about this some more

Reply:
err that's not it

```diff
             if isinstance(node.func, ast.Name):
+                import ipdb; ipdb.set_trace()
                 _replace_os_error(i, node.func)
```

```
ipdb> import astpretty                                                                                                                                            
ipdb> astpretty.pprint(node)                                                                                                                                      
Call(
    lineno=1,
    col_offset=6,
    func=Name(lineno=1, col_offset=6, id='EnvironmentError', ctx=Load()),
    args=[
        Num(lineno=1, col_offset=23, n=1),
        Num(lineno=1, col_offset=26, n=2),
    ],
    keywords=[],
)
```

This is triggered by the `main` test which is just `raise EnvironmentError(1, 2)`.